lib/app/widgets/error_widgets.dart

```diff
@@ -41,7 +41,7 @@ class _MessageAndActions {
       message = s.app_errorScreen_authError;
       actions.add(SecondaryButton(
         onPressed: () => logOut(context),
-        child: Text(s.app_errorScreen_authError_logOut),
+        child: Text(s.app_errorScreen_authError),
```

Comment:
This text seems to be quite long for a simple button… Did you mean general_signOut?

Reply:
Yes, I meant
```suggestion
        child: Text(s.general_signOut),
```